tests/plugins/link/manual/keystroke.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: feature, link, 2478, 4.11.0
+@bender-ckeditor-plugins: link, toolbar, wysiwygarea
+@bender-ui: collapsed
+----
+
+1. Focus the editor.
+2. Press <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>K</kbd>.
+
+	## Expected
```

Comment:
I'm not really sure about test construction. I'm more used to linear tests. Although it's subjective, so would like to see the opinion of another reviewer.